lib/beaker/dsl/helpers/host_helpers.rb

```diff
@@ -254,15 +254,27 @@ def rsync_to host, from_path, to_path, opts = {}
         # @option [String] archive_root The local directory to store the copied
         #                               file under. Defaults to
         #                               'archive/sut-files'.
+        # @option [String] archive_name The name of the archive to be copied to
+        #                               archive_root. Defaults to
+        #                               'sut-files.tgz'.
         #
         # @return [Result] Returns the result of the #scp_from operation.
-        def archive_file_from(host, from_path, opts = {}, archive_root = 'archive/sut-files')
+        def archive_file_from(host, from_path, opts = {}, archive_root = 'archive/sut-files', archive_name = 'sut-files.tgz')
+          require 'archive/tar/minitar'
           filedir = File.dirname(from_path)
           targetdir = File.join(archive_root, host.hostname, filedir)
           FileUtils.mkdir_p(targetdir)
           scp_from(host, from_path, targetdir, opts)
+          create_tarball(archive_root, archive_name)
         end
 
+        # @visibility private
```

Comment:
I'm not sure what this comment does; I think that instead you want to mark the method private by calling the private method and passing a symbol with the method name.

```ruby
module Foo

  def bar
  end
  private :bar

end

````